### PR TITLE
Disaster recovery without network restore

### PR DESCRIPTION
**Pull request by gsanchietti**
NethServer/dev#6099

### CODE REVIEW

File: administrator-manual/en/disaster_recovery.rst [hunk 1]
```diff
@@ -165,3 +168,104 @@ the configuration at boot-time, to ensure a minimal network
 connectivity and login again on the Server Manager.
 
 
+.. _skip-network-restore-section:
+
+Skip network restore
+--------------------
+
+Network configuration is restored by default, but sometimes it's necessary to restore an
```
Comment (DavidePrincipi) [inline administrator-manual/en/disaster_recovery.rst:176]: Do not use contracted forms

```suggestion
Network configuration is restored by default, but sometimes it is necessary to restore an 
```

File: administrator-manual/en/disaster_recovery.rst [hunk 2]
```diff
@@ -165,3 +168,104 @@ the configuration at boot-time, to ensure a minimal network
...
+
+To disable the network restore, make sure to disable the :guilabel:`Restore network configuration` option from
+the new Server Manager.
+
+Since some application configurations depend on network interface names, not everything can be automatically restored.
+
+DHCP
+^^^^
+
+DHCP servers on non-existing interface will be deleted.
```
Comment (DavidePrincipi) [inline administrator-manual/en/disaster_recovery.rst:188]:
```suggestion
DHCP servers on non-existing interfaces will be deleted.
```

File: administrator-manual/en/disaster_recovery.rst [hunk 3]
```diff
@@ -165,3 +168,104 @@ the configuration at boot-time, to ensure a minimal network
...
+If needed, please reconfigure the DHCP from the Server Manager.
+See also :ref:`dhcp-section` for more general information.
+
+Samba Active Directory
+^^^^^^^^^^^^^^^^^^^^^^
+
+Samba Active Directory requires a network bridge for the local running container.
```
Comment (DavidePrincipi) [inline administrator-manual/en/disaster_recovery.rst:195]:
```suggestion
Samba Active Directory requires a network bridge in the green zone for the local running container.
```

File: administrator-manual/en/disaster_recovery.rst [hunk 4]
```diff
@@ -165,3 +168,104 @@ the configuration at boot-time, to ensure a minimal network
...
+If the bridge already exists, the container will continue running after the restore.
+
+If the bridge doesn't exists anymore, Samba Active Directory is stopped.
```
Comment (DavidePrincipi) [inline administrator-manual/en/disaster_recovery.rst:198]:
```suggestion
If the bridge does not exist anymore, Samba Active Directory is stopped.
```

File: administrator-manual/en/disaster_recovery.rst [hunk 5]
```diff
@@ -165,3 +168,104 @@ the configuration at boot-time, to ensure a minimal network
...
+
+Only IPSec tunnels configured with a dynamic red interface will be disabled.
+Access the Server Manager, edit the disabled tunnel by selecting a new red interface and enable it again.
+
+More info at :ref:`ipsec-section`.
+
+Dedalo hotspot
+^^^^^^^^^^^^^^
+
+Dedalo hotspot will be disabled if the system doesn't have a network interface configured with the hotspot role.
```
Comment (DavidePrincipi) [inline administrator-manual/en/disaster_recovery.rst:260]:
```suggestion
Dedalo hotspot will be disabled if the system does not have a network interface configured with the ``hotspot`` role.
```

File: administrator-manual/en/disaster_recovery.rst [hunk 6]
```diff
@@ -165,3 +168,104 @@ the configuration at boot-time, to ensure a minimal network
...
+If the Dedalo is disabled, just reconfigure following :ref:`dedalo-section` chapter.
+
+ntopng
+^^^^^^
+
+ntopng must be reconfigured. Access the :guilabel:`Bandwidth monitor` page inside old Server Manager.
```
Comment (DavidePrincipi) [inline administrator-manual/en/disaster_recovery.rst:266]:
```suggestion
ntopng must be reconfigured. Access the :guilabel:`Bandwidth monitor` page inside the old Server Manager.
```

File: administrator-manual/en/disaster_recovery.rst [hunk 7]
```diff
@@ -165,3 +168,102 @@ the configuration at boot-time, to ensure a minimal network
...
+
+Samba Active Directory
+^^^^^^^^^^^^^^^^^^^^^^
+
+Samba Active Directory requires a network bridge in the green zone for the local running container.
+If the bridge already exists, the container will continue running after the restore.
+
+If the bridge does not exist anymore, Samba Active Directory is stopped.
+To enable it again:
+
```
Comment (gsanchietti) [inline administrator-manual/en/disaster_recovery.rst:200]:
```suggestion
- make sure the old IP address of the container is still inside the new green network. If not, set it from command line, eg: ::
    config setprop nsdc bridge IpAddress 192.168.1.2
```

Comment (DavidePrincipi) [reply administrator-manual/en/disaster_recovery.rst:200]: I want to make some tests: I'm not sure the nethserver-dc-save event is good. We may use nethserver-dc-change-ip instead.

**Comment by gsanchietti** [comment]
See also #489

**Pull request merged**